service-base/src/main/java/org/eclipse/hono/service/metric/MicrometerBasedMetrics.java

```diff
@@ -116,6 +126,10 @@ protected MicrometerBasedMetrics(final MeterRegistry registry, final Vertx vertx
             }
         });
         this.unauthenticatedConnections = registry.gauge(METER_CONNECTIONS_UNAUTHENTICATED, new AtomicLong());
+
+        this.rejectedConnections = Counter.builder(METER_CONNECTIONS_ATTEMPTS)
+            .tag(METER_CONNECTION_ATTEMPTS_TAG_KEY_OUTCOME, "adapter-connection-limit-exceeded")
+            .register(registry);
```

Comment:
this is only capable of recording a rejected connection attempt. What about the successful attempts?

Reply:
Right. Since we do not record successful attempts as of now, I went for the most simple solution.

But I took the chance to look at how this was done for the other metrics and adapted the implementation to match the approach of those. It's a bit more lines to read and understand but should be consistent now, see: 
df1620b